integration-tests/src/tests/client/process_blocks.rs

```diff
@@ -2121,6 +2121,7 @@ fn test_sync_hash_validity() {
 }
 
 /// Only process one block per height
+#[ignore]
```

Comment:
Can we fix this test? (or create another test that would verify that we will not forward a block in such scenario)

Reply:
I'm going to fix this test in another PR, where I will move all the process block related code in client actor to client, but that's a bigger change and I want to separate it